tepeszCodes/document-management-system
# Chore: setup CI and CD

#### What does this PR do?
Sets up CI with circleCI, CD with circleCI and heroku. It also implements code coverage check using coveralls and readme badges.
#### How should this be manually tested?
CD setup can't be tested manually. Only new scripts (`start` and `coveralls`) can be tested manually, run them in your console to test them. You can also check that the app is [deployed here](https://dms-express.herokuapp.com/).
#### Any background context you want to provide?
My TravisCI free trial has ended, so I used CircleCI instead
#### Relevant screenshots:
![coveralls screenshot](https://user-images.githubusercontent.com/54599709/139490918-2300a7ab-6314-4926-9d87-333aeb16b583.png)
![circleci screenshot](https://user-images.githubusercontent.com/54599709/139490925-edc28da6-3b29-4775-bbae-280812e7b524.PNG)
#### Questions:
I have some questions about environment variables, test coverage reports and CD pipelines, but I'll ask those questions during our meeting :)
